src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -621,14 +628,24 @@ bool BaseCouplingScheme::measureConvergence()
 {
   PRECICE_TRACE();
   PRECICE_ASSERT(not doesFirstStep());
-  bool allConverged = true;
-  bool oneSuffices  = false; // at least one convergence measure suffices and did converge
-  bool oneStrict    = false; // at least one convergence measure is strict and did not converge
-  PRECICE_ASSERT(_convergenceMeasures.size() > 0);
+  const bool reachedMinIterations = _iterations >= _minIterations;
   if (not utils::IntraComm::isSecondary()) {
     _convergenceWriter->writeData("TimeWindow", _timeWindows - 1);
     _convergenceWriter->writeData("Iteration", _iterations);
+    _convergenceWriter->writeData("ReachedMinIter", reachedMinIterations ? 1 : 0);
```

Comment:
Do we really need this additional information in the output? I think this part is somehow trivial.

Reply:
Trivial yes, but I would say also say it's useful for the reason of missing context.

Without `ReachedMinIter`, the file may state that everything has converged, yet the scheme continues to iterate. So, one needs to have a look at the configuration file to get the full picture.

Also, from the perspective of support, the convergence file now contains the information that the user configured a minimum of n iterations. So, the user only needs to provide one file.

An alternative to the boolean `ReachedMinIter` would be to simply add the `Min` value as a constant column. So the first columns of the file would look like this:

```
0 1 3
0 2 3
0 3 3
0 4 3
1 1 3
1 2 3
1 3 3
2 1 3
...
```

Reply:
After @uekerman voiced the same concerns, I removed the column.